apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProviderIT.java

```diff
@@ -1126,13 +1125,13 @@ public void searchForExistingPatientByMbiHashIncludeIdentifiersTrue() {
   /**
    * Verifies that {@link PatientResourceProvider#searchByIdentifier} returns the historical MBI
    * values in the response when searching by historic (non-current) MBI hash. The search should
-   * look in the medicare_beneficiaryid_history for historical MBIs to include in the response when
-   * IncludeIdentifiers is set to true in the header.
+   * look in beneficiaries_history for historical MBIs to include in the response.
    *
```

Comment:
The second part of this should still be true for v1 (only check historical if includeIdentifiers is in the header)

Reply:
done.